notebooks/cognee_demo.ipynb

```diff
@@ -667,6 +669,10 @@
                 "                    summarization_model = cognee_config.summarization_model,\n",
                 "                    collection_name = \"summaries\",\n",
                 "                ),\n",
+                "             Task(\n",
```

Comment:
Does this task work as expected? We didn't use it for a while.

Reply:
I don't know if it gives the expected output, but it doesn't cause problems anymore for the notebook and tests regarding UUIDs due to fixing the issue with cleaning the database. If the database would not be cleaned it would cause these issues though. The Task is needed for the insights search, otherwise it complains that it doesn't have a classification table.

Reply:
Seems fine then, no obvious error, we can add tests for it later and figure out how to test each task